.travis.yml

```diff
@@ -10,9 +10,14 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
+before_install:
+  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
 env:
   - KOREBUILD_DNU_RESTORE_CORECLR=true KOREBUILD_TEST_DNXCORE=true
```

Comment:
@kichalla you're correct that `MONO_THREADS_PER_CPU` doesn't matter for `dnu restore` anymore. Only need to set this in `.travis.yml` if testing sometimes hits threading issues. Fairly obvious that's not the case here -- tests haven't been terrible.